### PR TITLE
Add ROADMAP.md to the project and cleanup existing items

### PR DESCRIPTION
**Pull request by stevvooe**
Signed-off-by: Stephen J Day <stephen.day@docker.com>

### CODE REVIEW

**Comment by stevvooe** [comment]
cc @amylindburg @dmp42

**Comment by amylindburg** [comment]
The target date for Platform 1.8 is probably sooner -- convention is the first Tuesday in (every other) month -- so this would be August 4th. Let me get the project pages live ... I will send you the links...

**Comment by amylindburg** [comment]
Okay -- the wiki is live. Here are the links to the project pages:
https://github.com/docker/distribution/wiki/docker-1.6-Project-Page
https://github.com/docker/distribution/wiki/docker-1.7-Project-Page
https://github.com/docker/distribution/wiki/docker-1.8-Project-Page

There are "starter" goals in 1.7 and 1.8; we need to fill out the Release Goals (AKA Release Notes) for 1.6

**Comment by amylindburg** [comment]
I changed the links in the Wiki to point to this ROADMAP.md (when it is merged).

**Comment by dmp42** [comment]
LGTM as a good first step. We can refine later.

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+***
+
+### Distribution Package 
+
+At its core, the Distribution Project is a set of Go packages that make up
+Distribution Components. At this time, most of these packages make up the
+Registry implementation. 
+
+The package itself is considered unstable. If you're using it, please take care to vendor the dependent version.
```
Comment (SvenDowideit) [inline ROADMAP.md]: do you mean that the `master` package is considered unstable? otherwise it reads as though this codebase is not ready for going live.

**Comment by SvenDowideit** [comment]
LGTM - there's minor stuff - like a few lines that are way more than 80 chars.

File: ROADMAP.md (hunk 1 above)
Comment (stevvooe) [inline ROADMAP.md]: This is referring to the package API. For example, we might change names and method signatures.

**Comment by stevvooe** [comment]
@SvenDowideit The only lines over 80 characters are those with URLs. Is that an issue?

**Comment by SvenDowideit** [comment]
not imo :)

**Comment by stevvooe** [comment]
That is the only one I didn't check. Please submit a PR for formatting, if you are so inclined.

**Pull request merged**